resources/js/pages/admin/phrase-panel/create/NewPhraseForm.jsx

```diff
@@ -52,6 +56,14 @@ export default function NewPhraseForm() {
     if (!success) {
       return setErrors(errors)
     }
+    setLanguages(languages)
+    setValues({
+      phrase: '',
+      topic: '',
+      translations: languages ? new Array(languages.length).fill('') : []
```

Comment:
This needs to be reviewed - we cannot provide empty fields once we create the phrase.... it keeps the old values